config/routes/Centreon/monitoring/comments.yaml

```diff
@@ -0,0 +1,22 @@
+centreon_application_monitoring_commentresource:
+    methods: POST
```

Comment:
in the description the verb is PUT

Reply:
I @sc979 I do not understand your remark :smiley_cat:

Reply:
Modified :+1: :100: